docs/running.rst

```diff
@@ -231,3 +235,110 @@ These options are only accepted by the Text User Interface.
   Set the path of the user preferences file to use, which might override some of the options above.
   Specifying userPrefs as first argument would mean some later options will override them, as last argument would mean they will override some previous options).
   This rationale behind this option is to reuse SpotBugs Eclipse project settings for command line execution.
+
+-showPlugins:
+  Show list of available detector plugins.
+  
+-pluginList <jar1[;jar2...]>:
+  Specify list of plugin Jar files to load.
+  
+-home <home directory>:
+  Specify SpotBugs home directory.
+    
+-adjustExperimental:
+  Lower priority of experimental Bug Patterns.
+  
+-workHard:
+  Ensure analysis effort is at least 'default'.
+
+-conserveSpace:
+  Same as -effort:min (for backward compatibility).
+
+Output options
+^^^^^^^^^^^^^^
+-timestampNow:
+  Set timestamp of results to be current time.
+
+-quiet:
+  Suppress error messages.
+
+-longBugCodes:
+  Report long bug codes.
+
+-progress:
+  Display progress in terminal window.
+
+-release <release name>:
+  Set the release name of the analyzed application.
+
+-maxRank <rank>:
+  Only report issues with a bug rank at least as scary as that provided.
+
+-dontCombineWarnings:
+  Don't combine warnings that differ only in line number.
+
+-train[:outputDir]:
```

Comment:
maybe `train:[outputDir]` or `train <outputDir>`?

Reply:
It <is> optional, I found it [on the source].

```
class edu.umd.cs.findbugs.TextUICommandLine:
addSwitchWithOptionalExtraPart("-train", "outputDir", "Save training data (experimental); output dir defaults to '.'");
```

Reply:
So `[:foo]` means optional parameter? It's first time for me to see this expression...

Reply:
The The braces [ ] to indicate "zero or more", https://www.techrepublic.com/blog/software-engineer/what-is-bnf-and-why-should-developers-care 
Not sure about the colon : meaning yet, but guessing it is a variable indicator in some sort of notation.

Reply:
OK I see, you copied output from that command line help, then I can understand its meaning.

That `:` means necessary separator. For instance, for `-train[:outputDir]` means that we need to input like `-train:/tmp/foo`.